README.md

```diff
@@ -103,30 +103,39 @@ yarn upgrade @scientist-softserv/webstore-component-library --latest
 Using the local github repository requires you to manually clone the component library to your computer, build, and link it:
 
 #### Preparing your local copy of the component library:
-_Prerequisite: clone the [webstore-component-library](https://github.com/scientist-softserv/webstore-component-library.git) and [get the app running](https://github.com/scientist-softserv/webstore-component-library#running-the-app)_
-
-    cd webstore-component-library
-    yarn link # now there is a magic symlink in `~/.config/yarn/link` usable by the webstore app
+_Prerequisite: clone the [webstore-component-library](https://github.com/scientist-softserv/webstore-component-library.git) (wcl) and [get the app running](https://github.com/scientist-softserv/webstore-component-library#running-the-app)_
+``` bash
```

Comment:
wrapped the code blocks in backticks so the commenting happened correctly. same as below.